scripting/shavit-zones.sp

```diff
@@ -642,6 +660,65 @@ public Action Command_TPZone(int client, int args)
 	return Plugin_Handled;
 }
 
+public Action Command_AddSpawn(int client, int args)
+{
+	if(!IsValidClient(client))
+	{
+		return Plugin_Handled;
+	}
+
+	if(!IsPlayerAlive(client))
+	{
+		ReplyToCommand(client, "%s You can't place zones when you're dead.", PREFIX);
+
+		return Plugin_Handled;
+	}
+
+	gMZ_Type[client] = Zone_CustomSpawn;
+	GetClientAbsOrigin(client, gV_Point1[client]);
+
+	if(gB_CustomSpawn)
```

Comment:
Not sure what you mean? This is part of Command_Addspawn, also you're looking at an older commit?